Week1/homework/app.js

```diff
@@ -1,11 +1,169 @@
+  // another object nested inside books object, inside that object I have four keys and values:
+  // Here I wanted to create a constructor function! But I didn't want to get out of question context.
+  // cover: link to the photo alls sources from wikipedia.
+  // title of the book.
+  // the name of the author.
+  // book's language.
+
+  const books = {
+    'the-da-vinci-code': {
+      title: 'The Da Vinci Code',
+      cover: 'https://upload.wikimedia.org/wikipedia/en/6/6b/DaVinciCode.jpg',
+      language: 'English',
+      author: 'By Dan Brown',
+    },
+    'the-hunger-games': {
+      title: 'The Hunger Game',
+      cover: 'https://upload.wikimedia.org/wikipedia/en/d/dc/The_Hunger_Games.jpg',
+      language: 'English',
+      author: 'By Suzanne Collins',
+    },
+    'the-maze-runner': {
+      title: 'The Maze Runner',
+      cover: 'https://upload.wikimedia.org/wikipedia/en/d/db/The_Maze_Runner_cover.png',
+      language: 'English',
+      author: 'By James Dashner',
+    },
+    'the-lord-of-the-rings': {
+      title: 'The Lord of the Rings',
+      cover:
+        'https://upload.wikimedia.org/wikipedia/en/e/e9/First_Single_Volume_Edition_of_The_Lord_of_the_Rings.gif',
+      language: 'English',
+      author: 'By J. R. R. Tolkien',
+    },
+    'the-hobbit': {
+      title: 'The Hobbit',
+      cover: 'https://upload.wikimedia.org/wikipedia/en/4/4a/TheHobbit_FirstEdition.jpg',
+      language: 'English',
+      author: 'By J. R. R. Tolkien',
+    },
+    'a-game-of-thrones': {
+      title: 'A Game of Thrones',
+      cover: 'https://upload.wikimedia.org/wikipedia/en/9/93/AGameOfThrones.jpg',
+      language: 'English',
+      author: 'By George R. R. Martin',
+    },
+    'harry-potter-and-the-goblet-of-fire': {
+      title: 'Harry Potter and the Goblet of Fire',
+      cover:
+        'https://upload.wikimedia.org/wikipedia/en/c/c7/Harry_Potter_and_the_Goblet_of_Fire.jpg',
+      language: 'English',
+      author: 'By J. K. Rowling',
+    },
+    'the-book-thief': {
+      title: 'The Book Thief',
+      cover:
+        'https://upload.wikimedia.org/wikipedia/en/8/8f/The_Book_Thief_by_Markus_Zusak_book_cover.jpg',
+      language: 'English',
+      author: 'By Markus Zusak',
+    },
+    'angels-and-demons': {
+      title: 'Angels and Demons',
+      cover: 'https://upload.wikimedia.org/wikipedia/en/5/5f/AngelsAndDemons.jpg',
+      language: 'English',
+      author: 'By Dan Brown',
+    },
+    'harry-potter-and-the-order-of-the-phoenix': {
+      title: 'Harry Potter and the Order of the Phoenix',
+      cover:
+        'https://upload.wikimedia.org/wikipedia/en/7/70/Harry_Potter_and_the_Order_of_the_Phoenix.jpg',
+      language: 'English',
+      author: 'By J. K. Rowling',
+    },
+  };
+
+  // I created another function, which takes books object as a parameter  Object.keys() method returns an array of a given object's own property names
```

Comment:
I think the comments here again are a bit too much. You can avoid having such long comments in the future assignments. For example, if you name the parameter of your function as `books` instead of `obj`, you wouldn't need the comment `function which takes books object as a parameter`